src/test/java/math/Matrix4x4Test.java

```diff
@@ -0,0 +1,51 @@
+package math;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class Matrix4x4Test {
```

Comment:
`Matrix4x4` has other methods too, like - `zero`, `setCell` `negate` and others. Add tests for all of them.